src/test/java/io/aiven/connect/jdbc/sink/BufferedRecordsTest.java

```diff
@@ -39,20 +40,27 @@
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
-import org.mockito.Matchers;
-import org.mockito.Mockito;
+import org.mockito.ArgumentCaptor;
 
+import static java.sql.Statement.SUCCESS_NO_INFO;
 import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Matchers.eq;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
 public class BufferedRecordsTest {
 
     private final SqliteHelper sqliteHelper = new SqliteHelper(getClass().getSimpleName());
+    private String dbUrl;
```

Comment:
Could be `final`?

Reply:
Hmm, now that you ask, I think yes. I put it into setUp(), since I thought `sqliteHelper.sqliteUri()` can be called only after `sqliteHelper ` is actually set up, but that doesn't appear to be the case.